database/factories/CitationFactory.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Citation;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Citation>
+ */
+class CitationFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Citation::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'doi' => null,
+            'title' => $this->faker->title(),
+            'authors' => $this->faker->firstName().$this->faker->lastName(),
```

Comment:
If the author's field expects a string of author names, please include at least two and concatenate the strings following one standard approach (refer to the place holder text).